optaplanner-examples/src/main/java/org/optaplanner/examples/scrabble/domain/ScrabbleCell.java

```diff
@@ -36,7 +36,7 @@
 
     @CustomShadowVariable(variableListenerClass = CellUpdatingVariableListener.class,
             sources = {@PlanningVariableReference(entityClass = ScrabbleWordAssignment.class, variableName = "startCell"),
-                    @PlanningVariableReference(entityClass = ScrabbleWordAssignment.class, variableName = "direction"),})
+                       @PlanningVariableReference(entityClass = ScrabbleWordAssignment.class, variableName = "direction")})
```

Comment:
@bradh If you're using IntelliJ or Eclipse you can import these project-wide [config files](https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/tree/master/ide-configuration) and have the IDE format it for you